influxdb3/src/commands/create.rs

```diff
@@ -0,0 +1,327 @@
+use crate::commands::common::{parse_key_val, DataType, InfluxDb3Config, SeparatedList};
+use base64::engine::general_purpose::URL_SAFE_NO_PAD as B64;
+use base64::Engine as _;
+use rand::rngs::OsRng;
+use rand::RngCore;
+use secrecy::ExposeSecret;
+use secrecy::Secret;
+use sha2::Digest;
+use sha2::Sha512;
+use std::error::Error;
+use std::num::NonZeroUsize;
+use std::str;
+use url::Url;
+
+#[derive(Debug, clap::Parser)]
+pub struct Config {
+    #[clap(subcommand)]
+    cmd: SubCommand,
+}
+
+#[derive(Debug, clap::Subcommand)]
+pub enum SubCommand {
+    /// Create a new database
+    Database(DatabaseConfig),
+    /// Create a new table in a database
+    Table(TableConfig),
+    /// Create a new auth token
+    Token,
+    /// Create a new last cache
+    LastCache(LastCacheConfig),
```

Comment:
Is it possible to have the subcommand use `_` instead of `-` so it is `last_cache`? See https://influxdata.slack.com/archives/C084G9LR2HL/p1735952132093929?thread_ts=1735920567.459899&cid=C084G9LR2HL